encrypted-config-value-module/src/main/java/com/palantir/config/crypto/jackson/JsonNodeStringReplacer.java

```diff
@@ -29,13 +29,12 @@
 import com.palantir.config.crypto.util.StringSubstitutionException;
 import java.util.Iterator;
 import java.util.Map;
-import org.apache.commons.lang3.text.StrSubstitutor;
 
 public final class JsonNodeStringReplacer implements JsonNodeVisitor<JsonNode> {
 
-    private final StrSubstitutor substitutor;
+    private final Substitutor substitutor;
 
-    public JsonNodeStringReplacer(StrSubstitutor substitutor) {
+    public JsonNodeStringReplacer(Substitutor substitutor) {
```

Comment:
this will be a Java compile break, though now we're defining our own `Substitutor` functional interface